### PR TITLE
Anvil resource guidance

### PR DESCRIPTION
**Pull request by dglass19**

### CODE REVIEW

**Comment by dglass19** [comment]
@mdw333 @kevinamstutz @dgc-purdue I'd love to get your thoughts on the new section under Starter Guides -> Anvil -> Anvil Resource Guide. My goal is to help students and mentors know what's available to them, but also how we need to be aware of the amount of resources that we are using.

File: starter-guides/modules/anvil/pages/anvil-resources.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+= Anvil Resource Guide
+
+Purdue's Anvil high performance computing (HPC) cluster allows users to take advantage of world-class resources for use in their experiential learning projects. Anvil contains both CPU and GPU resources that are available to student teams. 
+
+The sections below are designed to help users understand the resources that are available on Anvil and the different request processes to ensure that The Data Mine can provide the tools needed for the experiential learning projects. 
+
+== Requesting GPU Resources
+
+By default, students are only given access to the CPU resources on Anvil. This is due to two factors. One is that Anvil is designed as a primarily CPU first system. Due to this, it has less GPU resources available when compared with other HPC clusters that are designed for primarily GPU usage. The second is that GPU resources have a higher associated cost because of their demand and technological advancement.
```
Comment (kevinamstutz) [inline starter-guides/modules/anvil/pages/anvil-resources.adoc:9]: You could put:

> The second is that GPU resources utilize credits at a rate 66 times greater than CPU.

Or something like that.

File: starter-guides/modules/anvil/pages/anvil-resources.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+If you know that your project is going to require a large amount of resources (memory, storage, processing, etc.) please reach out to The Data Mine at datamine-help@purdue.edu before starting anything. Please include your team’s name and a brief overview of your use case in the ticket description. This allows our technical team to work with you to ensure that the resources are available without limiting the resources that are available to any of the other teams on the shared HPC environment. 
+
+== Additional Processing Power
+
+The Jupyter Notebook sessions that The Data Mine provides have a limited number of cores available to them at launch. This is by design to ensure that we are good stewards of the HPC resources. However, we do recognize that teams will need the ability to launch sessions with larger number of cores in special situations.
```
Comment (kevinamstutz) [inline starter-guides/modules/anvil/pages/anvil-resources.adoc:21]: You could put:

> This is by design to ensure that we are good stewards of the HPC resources. In addition, its unlikely additional cores will improve performance unless you are utilizing specific libraries designed to bypass Python's GIL and/or launch multiple Python processes.

**Review by kevinamstutz — APPROVED** [review]
Looks good to me!

**Comment by dglass19** [comment]
> Looks good to me!

Thanks @kevinamstutz! I made the changes. Going to go ahead and merge in V1.

**Pull request merged**